repose-aggregator/src/docs/asciidoc/recipes/vagrant.adoc

```diff
@@ -0,0 +1,130 @@
+= Vagrant
+
+== Overview
+If you need a quick Linux environment setup to test or evaluate Repose, https://www.vagrantup.com/[Vagrant] makes it easy to create a new virtual machine with the https://atlas.hashicorp.com/boxes/search[Linux distribution of your choice] installed.
+
+== Installation
+=== VirtualBox
+Vagrant needs a virtual machine provider such as VirtualBox.
+https://www.virtualbox.org/[VirtualBox] can be installed on a variety of operating systems.
+Check out the https://www.virtualbox.org/wiki/Downloads[VirtualBox Downloads] page and install the binary for your operating system.
+
+=== Vagrant
+Check out the https://www.vagrantup.com/downloads.html[Vagrant Downloads] page and install the package for your operating system.
+
+=== Vagrant Plugins
+It is extremely useful to install the Vagrant plugin for the VirtualBox Guest Additions.
+To do so, run:
+
+[source,bash]
+----
+vagrant plugin install vagrant-vbguest
+----
+
+== Which way? Build Environment OR Manual?
+From here you can either install a few more tools and use the automated build environment that the Repose team uses to verify a release, or you can manually set up your environment.
+
+=== Build Environment
+==== Install Git
+Check out the https://git-scm.com/downloads[Git Downloads] page and install the package for your operating system.
+
+==== Install Gradle
```

Comment:
Installing Gradle isn't really necessary. The Gradle wrapper can be used.

The examples using Gradle should use gradlew instead, then.